assets/src/edit-story/utils/processPastedNodeList.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import escapeHTML from './escapeHTML';
+
+const ALLOWED_CONTENT_NODES = ['strong', 'em', 'u'];
+const TAG_REPLACEMENTS = {
+  b: 'strong',
+  i: 'em',
+};
+
+export default function processPastedNodeList(nodeList, content) {
+  for (let i = 0; i < nodeList.length; i++) {
+    const n = nodeList[i];
+    let tag = n.tagName?.toLowerCase();
+    tag = TAG_REPLACEMENTS[tag] ? TAG_REPLACEMENTS[tag] : tag;
```

Comment:
You can use the awesome [`??`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) here! Also I prefer to use two different `const`'s rather than a `let`:

```suggestion
    const originalTag = n.tagName?.toLowerCase();
    const tag = TAG_REPLACEMENTS[originalTag] ?? originalTag;
```

Reply:
Thanks! Changed.